src/applications/personalization/profile-2/components/AccountSecurityContent.jsx

```diff
@@ -69,7 +73,8 @@ export const AccountSecurityContent = ({
   return (
     <>
       {!isIdentityVerified && <IdentityNotVerified />}
-      {!isInMVI && isIdentityVerified && <NotInMVI />}
```

Comment:
the logic to determine if we show the `not in MPI` error got more complex with the inclusion of the new `can't connect to MPI` error; this is the main reason I moved the conditionals to `mapStateToProps`